src/mlpack/methods/decision_tree/split_functions/best_binary_numeric_split.hpp

```diff
@@ -13,7 +13,7 @@
 #define MLPACK_METHODS_DECISION_TREE_BEST_BINARY_NUMERIC_SPLIT_HPP
 
 #include <mlpack/prereqs.hpp>
-#include <mlpack/methods/decision_tree/fitness_functions/mse_gain.hpp>
+# include "../fitness_functions/mse_gain.hpp"
```

Comment:
This should not be a problem, therefore I do not see direct reason for the change, but anyway this is not a problem itself.

Reply:
It was a bit of an issue running locally, I'll change it back :+1: